contracts/l1/Bridge.sol

```diff
@@ -312,26 +321,29 @@ contract Bridge is IBridge, Initializable {
         (payload[5], payload[6]) = Cairo.toSplitUint(blockNumber);
         (payload[7], payload[8]) = Cairo.toSplitUint(currentRewardsIndex);
 
-        _messagingContract.sendMessageToL2(
+        uint256 nonce;
+        (, nonce) = _messagingContract.sendMessageToL2{value: fee}(
```

Comment:
In our opinion, there should be some hard limit of this fee ie, 0.1ETH, and revert in case a user sends more. Because, most likely, it can happen only in the case of some bug.

So I would wrap this logic into some internal function because it is used many times in a code.